drivers/tee/optee/supp.c

```diff
@@ -45,6 +45,7 @@ void optee_supp_uninit(struct optee_supp *supp)
 u32 optee_supp_thrd_req(struct tee_context *ctx, u32 func, size_t num_params,
 			struct tee_param *param)
 {
+	bool interruptable;
```

Comment:
The variable indicates whether we may interrupt here, we already know that someone is trying to interrupt. If there's a supplicant we shouldn't. But I'm inclined to change it to `interruptible` to harmonize better with the function names.